recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,4 @@
 processor:
   - cpu
-
+vc:
+- 14
```

Comment:
What's vc? Also, do there need to be spaces before the `-14` (similar to the `  - cpu` line)?

Reply:
Indeed, this should be removed, I needed it to pick up the right compiler on windows when building locally and I have committed it by mistake; it is removed now. Normally, it should make no difference on azure pipelines.

Reply:
cool. I guess we could merge it now then.